src/concrete/ml/sklearn/base.py

```diff
@@ -2034,7 +2035,7 @@ def predict(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.
         X = check_array_and_assert(X)
 
         topk_labels = []
-        for query in X:
+        for query in tqdm(X):
```

Comment:
let's avoid tqdm

Reply:
It was just for me, to have an idea about status of the execution.